xandikos/access.py

```diff
@@ -22,7 +22,6 @@
 See http://www.webdav.org/specs/rfc3744.html
 """
 
-from defusedxml.ElementTree import fromstring as xmlparse
```

Comment:
Cool, I wan't aware. If it does monkey-patching, won't doing it in `xandikos.__init__.py` suffice?